common/src/test/java/apoc/util/UtilIT.java

```diff
@@ -63,10 +57,26 @@ public void redirectShouldWorkWhenProtocolNotChangesWithUrlLocation() throws IOE
         assertTrue(page.contains("<title>Google</title>"));
     }
 
-    @Ignore
+    @Test
+    public void redirectWithBlockedIPsWithUrlLocation() {
```

Comment:
Is this url actually redirecting?

Reply:
setUpServer takes a redirect URL; in this case "http://127.168.0.1". Then getServerUrl() will call the Docker Containers IP (is assigned on startup. Then if the Dockers ip is called, the redirect command will happen, which returns a 301 followed by the input redirect URL, so it goes DOCKER_IP --> "http://127.168.0.1", the DOCKER_IP is not deny_listed so will pass, then the latter IP is deny_listed, so will fail